src/Domain/Person.php

```diff
@@ -68,6 +64,22 @@ class Person
      */
     public array $cash_balance = [];
 
+    /**
+     * @OA\Property(
+     *     type="object",
+     *     description="Properties are lowercase currency codes, e.g. 'gbp'. Values are
+     *     available amounts in smallest denomination, e.g. 123 pence.",
+     *     example={
+     *         "eur": 0,
+     *         "gbp": 123,
+     *     }
+     * )
+     * @var null|int[]  Pending Payment Intents for Big Give (i.e. donor fund top up tips)
```

Comment:
Let's make the type a bit more specific. This doesn't need any other changes, since the currency is already typed as string where we set this.

```suggestion
     * @var null|array<string,int>  Total Pending Payment Intents for Big Give (i.e. donor fund top up tips) for each currency
```